scripts/bootstrap.sh

```diff
@@ -44,6 +48,11 @@ install_devbox() {
 	echo "=== devbox installed..."
 }
 
+add_current_user_to_admin_group() {
+	echo "=== add current user to admin group"
+	sudo dseditgroup -o edit -a "$(whoami)" -t user admin
+}
+
```

Comment:
Does this have any impact outside of devbox set up?

Reply:
it's adding current user to group `admin`,  all CA managed users should already in that group so there won't be any issue